docs/src/main/asciidoc/spring-cloud-vault.adoc

```diff
@@ -650,6 +690,8 @@ See also: https://www.vaultproject.io/docs/secrets/cassandra/index.html[Vault Do
 [[vault.config.backends.mongodb]]
 === MongoDB
 
+This backend has been deprecated and might be removed from Vault soon.
```

Comment:
We should not deprecate this one because it maps to different properties. It's common to use a single JDBC database along with several NoSQL stores. Removing/deprecating removes the ability to plug into multiple databases with Spring Boot and Vault.

It would rather make sense to hint towards the database backend.